drone/format/format.go

```diff
@@ -26,22 +19,5 @@ var Command = cli.Command{
 }
 
 func format(c *cli.Context) error {
-	path := c.Args().First()
-	if path == "" {
-		path = ".drone.yml"
-	}
-
-	manifest, err := yaml.ParseFile(path)
-	if err != nil {
-		return err
-	}
-
-	buf := new(bytes.Buffer)
-	pretty.Print(buf, manifest)
-
-	if c.Bool("save") {
-		return ioutil.WriteFile(path, buf.Bytes(), 0644)
-	}
-	_, err = io.Copy(os.Stderr, buf)
-	return err
+	return nil
```

Comment:
I recommend removing the formatting code, and instead just output the input, unchanged

```
	if path == "" {
		path = ".drone.yml"
	}

	out, _ := ioutil.ReadFile(path)
	buf := bytes.NewBuffer(out)

	if c.Bool("save") {
		return ioutil.WriteFile(path, buf.Bytes(), 0644)
	}
	_, err = io.Copy(os.Stderr, buf)
	return err
```

This should prevent the change from breaking existing shell scripts that rely on the output of this command.

Reply:
done